Application/EdFi.Ods.Sandbox/Repositories/ClientAppRepo.cs

```diff
@@ -463,6 +481,7 @@ public Vendor CreateOrGetVendor(string userEmail, string userName, IEnumerable<s
                         }
                     }
 
+                    await context.SaveChangesAsync();
```

Comment:
This method didn't used to save, but it probably makes sense for it to ensure it saves after adding a new Vendor.  Shouldn't this Save call be inside the if check for "vendor == null" though?  No need to save if we're just retrieving an existing vendor.

Reply:
updated